qhub/schema.py

```diff
@@ -364,6 +364,27 @@ class Main(Base):
     environments: typing.Dict[str, CondaEnvironment]
     clearml: typing.Optional[ClearML]
 
+    @validator("project_name")
+    def project_name_convention(cls, value: typing.Any):
+        convention = """
+        In order to successfully deploy QHub, there are some project naming conventions which need
+        to be followed. For starters, make sure your name is compatible with the specific one for
+        your chosen Cloud provider. In addition, QHub projectname should also obey to the following
+        format requirements:
+        - letters from A to Z (upper and lower case) and numbers;
+        - Special characters are NOT allowed;
+        - Maximum accepted length of the name string is 16 characters.
+        - If using AWS: names SHOULD NOT start with the string aws;
+        """
+        if len(value) > 16:
+            raise ValueError(
+                "Maximum accepted length of the project name string is 16 characters."
+            )
+        elif re.findall(r"^(?!aws)[A-Za-z0-9][^\|\\?_-]*[A-Za-z0-9]$", value):
```

Comment:
```suggestion
        elif re.findall(r"^(?!aws)[A-Za-z0-9][^\|\\?_]*[A-Za-z0-9]$", value):
```
Dashes are acceptable in names. The previous regular expression looks for dashes and erroniously was throwing errors, causing all the tests to crash

Reply:
Thank you Tyler for the review, I red the special characters not allowed in the convention and thought that `-` was a problem to. Thank you, I will commit those changes.